### ISSUE TITLE
Please add more Symbols

### ISSUE THREAD

**Issue by arkhan** [belluzj/fantasque-sans#130]
Greetings,

Great work with this, it is my favorite typeface, I wish I could have some additional symbols as shown in the following images:

![2020-05-12-112801_1920x1080_scrot](https://user-images.githubusercontent.com/3141865/81722035-716fdf00-9446-11ea-92e0-4faf6f6c7652.png)

![2020-05-12-113614_1920x1080_scrot](https://user-images.githubusercontent.com/3141865/81722046-746acf80-9446-11ea-850a-b942cf0045ea.png)

Thank you

**Comment by patrickfatrick** [comment]
I know =>, <=, and >= are all included, you just need to include ligatures.

**Comment by arkhan** [comment]
@patrickfatrick Yes, I put that as an example, what I request is the symbols that are pixelated
